ERCS/erc-7715.md

```diff
@@ -87,23 +111,40 @@ Request example:
     type: 'key',
     data: {
       id:'did:key:zQ3shokFTS3brHcDQrn82RUDfCZESWL1ZdCEJwekUDPQiYBme',
-      }
     }
+  }
   permissions: [
     {
-      type: 'erc20-spending-limit',
-      data: {
-        erc20Address: '0x...',
-        limit: '0x...',
-      },
-      required: true
-    },
-    {
-      type: 'gas-limit',
+      type: 'erc20-transfer',
+      policies: [
+        {
+          type: 'token-allowance',
+          properties: {
+            allowance: 'uint256'
+          },
+          data: {
+            allowance: '0x...',
+          },
+        },
+        {
+          type: 'gas-limit',
+          properties: {
+            limit: 'uint256'
+          },
+          data: {
+            limit: '0x...',
```

Comment:
Do we really love hex everywhere? Would it be terrible to adopt decimal notation? Seems strictly easier to work with from a dx perspective, and the signer can always convert between whatever.